pkg/slack/client.go

```diff
@@ -302,15 +464,32 @@ func (c *Client) doRequest(ctx context.Context, url string, res interface{}, met
 
 	req.Header.Add("Accept", "application/json")
 	req.Header.Set("Content-Type", "application/x-www-form-urlencoded")
+
+	// different headers for SCIM API
+	if strings.Contains(url, baseScimUrl) {
```

Comment:
I know it'll probably never matter, but can we use HasPrefix instead of Contains?